> Initialized logger
> Exception in thread Malaria Detection:
> Traceback (most recent call last):
>   File "C:\Users\gherz-PC\Anaconda3\envs\py27\lib\threading.py", line 801, in __bootstrap_inner
>     self.run()
>  * Debugger is active!
>   File "C:\Users\gherz-PC\Desktop\remidiv2\serveus\detector.py", line 63, in run
>     self.query()
>   File "C:\Users\gherz-PC\Desktop\remidiv2\serveus\detector.py", line 67, in query
>     inf = Infection.query.filter(name=="Malaria").all()
> NameError: global name 'name' is not defined
>  * Debugger PIN: 456-115-780
>  * Running on http://0.0.0.0:80/ (Press CTRL+C to quit)
>  * Detected change in 'C:\\Users\\gherz-PC\\Desktop\\remidiv2\\serveus\\detector.py', reloading
